doc/api/v8.md

```diff
@@ -1,17 +1,29 @@
 # V8
 
-    Stability: 2 - Stable
+The `v8` module exposes APIs that are specific to the version of [V8][]
+built into the Node.js binary. It can be accessed using:
 
-This module exposes events and interfaces specific to the version of [V8][]
-built with Node.js.  These interfaces are subject to change by upstream and are
-therefore not covered under the stability index.
+```js
+const v8 = require('v8');
+```
+
+*Note*: The APIs and implementation are subject to change at any time.
```

Comment:
Given that this edit is a restatement of the existing notice, if we want to change this and declare that the v8 module *is* covered by the stability index, then we can do that in a separate PR.